### PR TITLE
 options-search: configure title 

### PR DESCRIPTION
**Pull request by MarcelCoding**
I've just added a parameter to the search that allows configuring the title. I hope the suggested one is what you like.

### CODE REVIEW

File: docs/default.nix [hunk 1]
```diff
@@ -72,6 +72,7 @@ in
       nuschtosSearch.packages.mkSearch {
         optionsJSON = options-json + "/share/doc/nixos/options.json";
         urlPrefix = "https://github.com/nix-community/nixvim/tree/main";
+        title = "NixVim Options Search";
```
Comment (GaetanLepage) [inline docs/default.nix:75]:
```suggestion
        title = "Nixvim options search";
```

Maybe this is more natural. I have no strong feeling though.

**Review by GaetanLepage — CHANGES_REQUESTED** [review]
Thanks for the change, this is a good idea.

Could you please get rid of the `flake.lock` update ? Is it necessary for this change ?

**Comment by MarcelCoding** [comment]
I've just committed the change into the search repo, as I need to update the search input in order that the `title` parameter exists.

PS: I've only updated the search input and left all others untouched.

**Comment by MarcelCoding** [comment]
(I've copied the *old* title from the current navigation sidebar.)
image

**Comment by GaetanLepage** [comment]
> I've just committed the change into the search repo, as I need to update the search input in order that the `title` parameter exists.

Ok I see. I have updated the lockfile in the meantime, so you should be able to rebase safely now.

**Comment by MarcelCoding** [comment]
done

**Review by GaetanLepage — APPROVED** [review]
LGTM ! Thanks for downstreaming this change to nixvim :)

**Comment by GaetanLepage** [comment]
@mergify queue

**Pull request merged**